integration/src/test/resources/org/mule/test/construct/flow-ref.xml

```diff
@@ -262,4 +262,12 @@
         <set-payload value="holis"></set-payload>
     </flow>
 
+    <flow name="outerFlow">
```

Comment:
rename to `outerFlowWithMaxConcurrency`

Reply:
Since it was a redundant test I decided to remove it in favor of the simpler, latter one.